debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -2674,7 +2941,7 @@ protected void assertSourceInfoMillisecondTransactionTimestamp(SourceRecord reco
         assertTrue(Math.abs(ts_ms - source.getInt64("ts_ms")) < tolerance_ms);
     }
 
-    private <T> void validateField(Config config, Field field, T expectedValue) {
+    private <T> void validateConfigField(Config config, Field field, T expectedValue) {
```

Comment:
Changed name for clarity, as there are two different `Field` classes referenced in this file (One for config, one for the Connect Schema)